src/Okta.Sdk.IntegrationTests/xunit.runner.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "parallelizeAssembly": true,
-  "parallelizeTestCollections": true,
+  "parallelizeAssembly": false,
+  "parallelizeTestCollections": false,
```

Comment:
Nice! Since it's possible to configure this via JSON for the entire project, should we remove the configuration we added recently in the code?

Reply:
I'm hesitant to do that because it seems the behavior is dependent on the runner used for a given run.